src/20230926_AaveV3_Eth_FundGHOLiquidityCommittee/FundGHOLiquidityCommittee.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Fund GHO Liquidity Committee"
+author: "TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-treasury-manage-gho-liquidity-committee/14914"
+---
+
+## Simple Summary
+
+This publication proposes the creation and funding of the GHO Liquidity Committee (GLC) for an initial 3-month period.
+
+## Motivation
+
+GHO is currently trading below its peg, with liquidity spread across various pools. To address this, the GLC will receive a budget of 5 ETH and 438,000 units of GHO with the primary objectives of:
+
+* Improving the GHO peg
+* Growing GHO liquidity
+
+The GLC's responsibilities include distributing rewards across liquidity pools and strategies that support GHO. The GLC is initially established for a 3-month period.
+
+## Specification
+
+This proposal encompasses the following actions:
+
+* Swap 438,000 aEthDAI to GHO via the Aave Swap Contract
```

Comment:
```suggestion
* Initiate the Swap 438,000 aEthDAI to GHO via the Aave Swap Contract
```

Reply:
also the aip sais 438k - but forum and impelemntation say 406k? and the conditions of the swap could be more elaborate here.